administration/src/bp-modules/cards/AddCardForm.tsx

```diff
@@ -32,11 +32,15 @@ const ExtensionForm = ({ extension, onUpdate }: ExtensionFormProps) => {
   })
 }
 
-const maxCardValidity = 99
-const hasCardExpirationError = (expirationDate: Date): boolean =>
-  isBefore(expirationDate, Date.now()) || isAfter(expirationDate, add(Date.now(), { years: maxCardValidity }))
+const maxCardValidity = { years: 99 }
+const hasCardExpirationError = (expirationDate: PlainDate): boolean => {
+  const today = PlainDate.fromLocalDate(new Date())
+
+  return expirationDate.isBefore(today) || expirationDate.isAfter(today.add(maxCardValidity))
+}
 
 const CreateCardForm = ({ cardBlueprint, onRemove, onUpdate }: CreateCardsFormProps) => {
+  const today = PlainDate.fromLocalDate(new Date())
```

Comment:
i think you could create `today` once before the `hasCardExpirationError` and also use it there since its just the date and not datetime

Reply:
I think it does not matter perf wise. Maybe some people are working during (mid-)night times, so we might only evalute `today` when we need it. (Or even worse: If you leave the tab open for a couple days, `today` would not change).